source/_components/sensor.swiss_hydrological_data.markdown

```diff
@@ -44,24 +46,46 @@ name:
   required: false
   type: string
   default: 
+measurements:
+  description: list of measurements you want to use.
+  required: true
+  type: list
+  deafult:
 {% endconfiguration %}
```

Comment:
This is a list and that should have a `keys` attribute. You can use that attribute to list the possible measurement values.